.github/workflows/node.js.yml

```diff
@@ -9,11 +9,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
-      - name: Use Node 16
-        uses: actions/setup-node@v3
+      - uses: actions/checkout@v4
+      - name: Use Node 20
+        uses: actions/setup-node@v4
         with:
-          node-version: 16
+          node-version: 20
```

Comment:
Please increase the node version in the `actions.yml` as well :)

Everything else looks great 👍🏼

Reply:
Done